README.md

```diff
@@ -1,12 +1,5 @@
-<p align="center">
-  <a href="https://hellofresh.com">
-    <img width="120" src="https://www.hellofresh.de/images/hellofresh/press/HelloFresh_Logo.png">
-  </a>
-</p>
-
 # hellofresh/stats-php
 
-[![Build Status](https://travis-ci.org/hellofresh/stats-php.svg?branch=master)](https://travis-ci.org/hellofresh/stats-php)
```

Comment:
We can replace this with github actions status badge (can be generated at https://github.com/hellofresh/stats-php/actions/workflows/testing.yml)

Reply:
Tbh I do not think we need this badge. Having tons of badges was cool 4 years ago =)